mlflow/store/artifact/utils/models.py

```diff
@@ -16,7 +18,10 @@ def _improper_model_uri_msg(uri):
     return (
         "Not a proper models:/ URI: %s. " % uri
         + "Models URIs must be of the form 'models:/<model_name>/suffix' "
-        + "where suffix is a model version, stage, or the string '%s'." % _MODELS_URI_SUFFIX_LATEST
+        + "or 'models:/<model_name>@alias' where suffix is a model version, stage, "
```

Comment:
```suggestion
        + "or 'models:/<model_name>@<alias>' where suffix is a model version, stage, "
```

`<>` to emphasize it represents a placeholder.